master/buildbot/spec/types/test_raw_result_chunk.raml

```diff
@@ -0,0 +1,31 @@
+#%RAML 1.0 DataType
+displayName: Test raw result chunk.
+description: |
+    This resource represents a part of test raw result.
+    All chunks for a sequence with the ``sequence`` value increasing sequentially starting from 0.
```

Comment:
not sure what this sequence number is about.

Reply:
This mimics the logchunk. In that case it contains the first line that the log chunk corresponds to. In the case of raw result chunk, we still want some way to identify specific result chunks.